app/models/import.rb

```diff
@@ -1,3 +1,7 @@
 class Import < ApplicationRecord
   belongs_to :importable, polymorphic: true, optional: true
+
+  def self.get_imported(entity, access_id)
+    find_by(importable_type: entity.to_s.classify, access_id: access_id)&.importable
```

Comment:
Why not pass the class (`entity`) directly instead of as a symbol?

Reply:
rails will convert that automaticly? Because in the db it will be saved as `'ClassName'`... ?

Reply:
Ah, I know what you mean.